gempy/core/data.py

```diff
@@ -6,6 +6,9 @@
 import numpy as np
 import pandas as pn
 
+from sklearn.neighbors import NearestNeighbors
```

Comment:
Can you move this into the function so gempy still works if people do not have installed sklearn

Reply:
Done!